src/test/java/com/google/cloud/tools/maven/it/RunMojoIntegrationTest.java

```diff
@@ -86,6 +97,34 @@ public void run() {
     assertEquals("Hello from the App Engine Standard project.", urlContent.toString());
     verifier.verifyErrorFreeLog();
     verifier.verifyTextInLog("Dev App Server is now running");
+    verifier.verifyTextInLog("Module instance " + expectedModuleName + " is running");
+  }
+
+  @SuppressWarnings("unused") // used for testRun()
+  private Object[] parametersForTestRun() {
+    List<Object[]> result = new ArrayList<>();
+    for (SupportedDevServerVersion serverVersion : SupportedDevServerVersion.values()) {
+      for (String[] profileAndServiceName : new String[][]{
+        { "", "standard-project"},
+        { "base-it-profile,appyamls", "standard-project-appyamls"},
+        { "base-it-profile,services", "standard-project-services"}
+        } ) {
+        result.add(
```

Comment:
I feel like I would prefer this code to just build the Object[] directly instead of through indirection via the String[][]
```
result.add(newObject[] {serverVersion, new String[]{"",""}, "standard-project"})
result.add(newObject[] {serverVersion, new String[]{"base-it-profile","appyamls"}, "standard-project-appyamls"})
result.add(newObject[] {serverVersion, new String[]{"base-it-profile","services"}, "standard-project-services"})
```
or maybe through a helper
```
paramHelperOrSomething(serverVersion, profile, subprofile, service)
```

whaddya think?